modules/social_features/social_activity/social_activity.tokens.inc

```diff
@@ -67,6 +69,28 @@ function social_activity_tokens($type, $tokens, array $data, array $options, Bub
                   ['absolute' => TRUE]
                 )->toString();
 
+                $string_parts = preg_split("/(\[\~\d+\])/", $summary, -1, PREG_SPLIT_DELIM_CAPTURE | PREG_SPLIT_NO_EMPTY);
```

Comment:
@albertoalaejos cc @navneet0693 

I don't think this is the proper way of dealing with this issue. most likely the mentions module will replace this (with input formats). So I don't think tokens is used for that.